js/start.js

```diff
@@ -307,6 +306,12 @@ function start() {
     app.ownFollowers = new Followers(null, { type: 'followers' });
 
     onboardIfNeeded().done(() => {
+      // add listener to pageNav after the profile is available
+      app.pageNav.listenTo(app.profile, 'change', () => {
```

Comment:
We can't get away with re-rendering here because the profile might be updated at random times (AJAX request completes at random time) and if the user is typing in the address field, it'll lose focus and whatever they are typing will be cleared. Also, the page url is lost on re-render. I don't think we'll be re-rendering this view too much.

I think the better approach here is to bind a change event specifically to the avatar field and then set the image url with jQuery.

Also, this event and handler should be bound in the PageNav view and not here.